### ISSUE TITLE
feat: Custom Controls & Multiple Decks

### ISSUE THREAD

**Issue by jtkeyva** [mkobuolys/flutter_deck#117]
**Description**

I want to modify the design and specifically add new links such as "exit" . I also want to add a "next deck" button in there.

**Requirements**

Customize design
Add custom buttons

**Additional Context**

Auto advance to next deck would be nice as well

**Comment by mkobuolys** [comment]
Hi, @jtkeyva could you elaborate on where you want to add these custom actions? You are talking about the presenter toolbar, correct?

**Comment by jtkeyva** [comment]
yes in the toolbar. wouldn't hurt to have an example of creating your own control/toolbar either

**Comment by mkobuolys** [comment]
> wouldn't hurt to have an example of creating your own control/toolbar either
1) Copy/paste the current code of the toolbar
2) Add your own controls
3) ???
4) Profit!

On a serious note, I will investigate the option of adding custom actions to the toolbar via the configuration, which could be a nice feature.

**Comment by jtkeyva** [comment]
1. Thanks
2. Sounds good
3. Multipe Decks!
4. Haha
